code/modules/reagents/reagent_containers/glass/beaker.dm

```diff
@@ -61,6 +61,17 @@
 	possible_transfer_amounts = list(5,10,15,25,30,60,120,300)
 	lid_icon_state = "lid_beakerlarge"
 
+/obj/item/weapon/reagent_containers/glass/beaker/bowl
+	name = "mixing bowl"
+	desc = "A large mixing bowl."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "mixingbowl"
+	matter = list(MATERIAL_STEEL = 2)
+	volume = 180
+	amount_per_transfer_from_this = 10	
+	possible_transfer_amounts = list(5,10,15,25,30,60,120,180)
+	unacidable = 0
```

Comment:
```suggestion
	unacidable = FALSE
```

Reply:
well i am 4 weeks in deployment so... if you can tell me how i can work this on here or without vs and smartgit i will change it

Reply:
nevermind, fixed